src/App.vue

```diff
@@ -37,71 +72,108 @@ export default {
   data() {
     return {
       showModal: false,
-    }
+    };
   },
   watch: {
-    theme(value) {
-      document.documentElement.classList.remove('dark', 'light', 'october')
-
-      switch(value) {
-        case 'dark':
-          document.documentElement.classList.add('dark')
-          break
-        case 'light':
-          document.documentElement.classList.add('light')
-          break
-        case 'october':
-          document.documentElement.classList.add('dark', 'october')
-          break
+    theme(value){
+      this.updateTheme()
```

Comment:
This `updateTheme` call looks correct now, but the switch/case statement right below this will need to be removed now that the same thing is happening in `updateTheme`.